app/views/curation_concerns/base/_form_state.html.erb

```diff
@@ -2,11 +2,16 @@
   <legend>State</legend>
   <div class="form-group">
```

Comment:
I can generate the radio buttons pretty easily, e.g.:

```
  <% states = { pending: 'In progress and suppressed from public display',
                review: 'In need of review',
                complete: 'Finished work, visibility determined by Access Rights' } %>
  <%= f.input :state, collection: states.map {|k,v| [k.to_s.titleize + ": " + v,k]}, as: :radio_buttons %>
```

But how do we generate the badges?